docs/getting-started-guides/minikube.md

```diff
@@ -0,0 +1,103 @@
+---
+---
+
+* TOC
+{:toc}
+
+Minikube starts a single node kubernetes cluster locally for purposes of development and testing.
+
+### Requirements
+
+Minikube requires that VT-x/AMD-v virtualization is enabled in BIOS on all platforms.
+
+To check that this is enabled on Linux, run:
+
+```shell
+cat /proc/cpuinfo | grep 'vmx\|svm'
+```
+
+This command should output something if the setting is enabled.
```

Comment:
Great.  Seems to be enable on my mac by default.  Any idea what mac versions this is enabled on by default?